Makefile.am

```diff
@@ -194,6 +196,41 @@ cc_oci_runtime_CFLAGS = \
 	$(LIBMNL_CFLAGS) \
 	$(UUID_CFLAGS)
 
+libexec_SCRIPTS = cc-proxy
+
+AM_V_GO    = $(am__v_GO_@AM_V@)
+am__v_GO_  = $(am__v_GO_@AM_DEFAULT_V@)
+am__v_GO_0 = @echo "  GOBUILD " $@;
+am__v_GO_1 =
+
+systemdservice_in_files =		\
+	proxy/cc-proxy.service.in	\
+	proxy/cc-proxy.socket.in	\
+	$(NULL)
```

Comment:
Removed it. If we need to document what it does because it's not clear enough as is, might as well not use it at all :)